modules/electrophysiology_browser/help/view_session.md

```diff
@@ -0,0 +1,15 @@
+# View Session
+
+This page displays all available information collected for an individual recording session.
+
+Summary information about the candidate, the recording session and the hardware used is displayed.
+
+Recording files can be downloaded individually or as a compressed folder (*All Files* download):
+
+- EEG: the file containing the session recording data.    
+- Electrode info (tsv): contains electrode locations.    
+- Channels info (tsv): channel status and filter settings.    
+- Events (tsv): events recorded during the session.   
+
+The compressed folder’s structure follows the BIDS standard for the EEG modality. More information about the BIDS standard can be found through their [website](https://bids.neuroimaging.io/).   
+On the left, *Next* and *Previous* buttons can be used to navigate between sessions by their number (the leftmost column in the front page of the module). Navigate back to the main page of the module by clicking on its name in the breadcrumb trail placed at the top of the page.
```

Comment:
```suggestion
On the left, *Next* and *Previous* buttons can be used to navigate between sessions. Navigate back to the main page of the module by clicking on its name in the breadcrumb trail placed at the top of the page.
```
removing a snipped, because that "number" either reflects the sort order, or acquisition/loading order (i.e. is meaningful)